### DISCUSSION TITLE
Can't install toltec on 2.14.1.866 (reMarkable 2)

### DISCUSSION THREAD

**Discussion by dphans** [toltec-dev/toltec#655] [Support] [answered]
I'm try to install tool by bootstrap file but found error, please check photo below:

```bash
wget http://toltec-dev.org/bootstrap
echo "04a28483286f88c5c7f39e352afb62adc57f6162a29fd7e124d832205bb0980e  bootstrap" | sha256sum -c && bash bootstrap
```

![Screen Shot 2022-08-20 at 20 49 20](https://user-images.githubusercontent.com/12996976/185749271-e3519b36-7dec-4245-b3ea-d6480d10384c.png)

**Comment by LinusCDE** [comment, marked as answer]
It seems that you had installed toltec previously. Usually the script would tell you, that you have toltec already installed, but it would not in a few cases:

 - toltecctl is missing (located at `/home/root/.local/bin/toltecctl`). Could mean a super old installation
 - `/opt` was cleaned but the mount wasn't removed (also checked `/home/root/.entware` but that would probably bind mounted together anyways)

Please run the following commands to make figuring this out more easy:

- `ls -la --full-time /home/root/.local/bin/toltecctl /home/root/.entware /opt` to check what files exist and if, how old they are
- `df -h` to confirm that /opt actually has no bind mount (as well as whether super low free space could be a potential issue as well)
- `ls -la --full-time /lib/systemd/system/opt.mount ; systemctl status opt.mount --no-pager` to check whether the service was already cleaned up

---

It seems someone also had this same issue a while back at https://github.com/toltec-dev/toltec/discussions/551. If you can run `toltecctl`, you can probably do a `toltecctl reenable` or `toltecctl uninstall` (backup any important config files in /opt first) to fix it. There was no real reason stated though, so this still needs to be investigated.
